README.md

```diff
@@ -1,32 +1,89 @@
+```
 
-    set TRY_SLANG_SOURCE_DIRECTORY_PATH=%userprofile%\source\try-slang
-    set TRY_SLANG_TARGET_DIRECTORY_PATH=%userprofile%\target\try-slang
+### Prerequisites
 
-In the following, it is also assumed that GNU Make, Python and gzip are in PATH.
-Follow the setup instructions in `docs/building.md` for building `slang-wasm` in the Slang repository.
-(E.g. setup shell for CMake and Emscripten, and setup the `build.em` output directory.)
+Ensure the following tools are installed and properly configured on your system:
+- **CMake** (Version 3.25 or greater is preferred, if you're on a lower version please see [build with older cmake](https://github.com/shader-slang/slang/blob/master/docs/building.md#building-with-an-older-cmake))
+- A **C++ compiler** with C++17 support (GCC, Clang, or MSVC)
+- A **CMake-compatible build backend** (e.g., Ninja or Visual Studio)
+- **Python 3** (Required for building and running the server)
+- **gzip** (For compressing `.wasm` files)
 
-### Server
+To install the above tools on linux, run:
+```bash
+sudo apt update
+sudo apt install build-essential cmake ninja-build python3 python3-pip gzip
+```
 
-The Wasm loader needs a running web server in the build target directory, in order to load .wasm files.
+We need to build `slang-wasm` because we need `slang-wasm.js` and `slang-wasm.wasm` files.
+The reason they're not included in this repo is that they are big binary files, and the result of building top of tree slang, so making this part of the CI build process makes the most sense.
 
-**NOTE: ** You do not need a shell with the full development environment set up, you just need Python and the `TRY_SLANG_TARGET_DIRECTORY_PATH` environment variable.
+### Building `slang-wasm`
 
-To start the server, run something like:
+1. Clone the [Slang repository](https://github.com/shader-slang/slang) and fetch its submodules:
+   ```bash
+   git clone https://github.com/shader-slang/slang --recursive
 
-    $ cd $TRY_SLANG_TARGET_DIRECTORY_PATH && python -m http.server 8000
+   cd ./slang
+   ```
 
-## Iterate
+2. Follow the instructions in the [WebAssembly build section](https://github.com/shader-slang/slang/blob/master/docs/building.md#webassembly-build) of the Slang documentation to:
+   - Set up the [Emscripten SDK](https://github.com/emscripten-core/emsdk) by installing and activating it.
+   - Build the WebAssembly target (`slang-wasm.js` and `slang-wasm.wasm`) using the documented cross-compilation steps.
 
-To iterate on the website, make changes and build the `website_runtime` target, from a development environment shell:
+3. Once the build completes, locate `slang-wasm.js` and `slang-wasm.wasm` in the `build.em/Release/bin` directory
+   
+4. Copy `slang-wasm.js` and `slang-wasm.wasm` to the **root of the playground directory**.
 
-    $ make -f $TRY_SLANG_SOURCE_DIRECTORY_PATH/build.md website_runtime
+5. Compress the `slang-wasm.wasm` file using gzip:
+   ```bash
+   gzip -k slang-wasm.wasm
+   ```
 
```

Comment:
Please also add the instruction of building spirv-tool wasm.
I left my comment here:
https://github.com/shader-slang/slang-playground/pull/78#issuecomment-2523676894

Reply:
Understood!